R/loadLm.R

```diff
@@ -88,7 +88,7 @@ loadLm <- function(formula, pred.format=c("flux","conc"),
                    data, metadata, fitting_function=NULL, 
                    y.trans.function=NULL, retrans.function=exp,
                    store=c("data","fitting.function"), ylog) {
-  
+  s.hat=NULL
```

Comment:
line 91 in loadLm.R; I removed this and the note for this section of the code no longer appears.